### PR TITLE
Add toggle for time

### PR DESCRIPTION
**Pull request by jewome62**
Add a preference to disable/enable time change from plot flag.

### CODE REVIEW

File: Nukkit/src/main/resources/plugin.yml [hunk 1]
```diff
@@ -119,6 +119,7 @@ permissions:
       plots.toggle: true
       plots.toggle.titles: true
       plots.toggle.chat: true
+      plots.toggle.time: true
```
Comment (N0tMyFaultOG) [inline Nukkit/src/main/resources/plugin.yml]: Is redundant, breaking doesn't build Nukkit artifacts :P

**Comment by MattBDev** [comment]
This whole idea seems half baked. How is this different from the current time flag that is already present?
Is this supposed to constantly keep the plot set to a current time all the time? If so, it fails to do it properly.

**Comment by AeSix** [comment]
@MattBDev  maybe something to do with https://github.com/IntellectualSites/PlotSquared/issues/2412
Is this correct @jewome62 ?

File: Nukkit/src/main/resources/plugin.yml (hunk 1 above)
Comment (Sauilitired) [reply Nukkit/src/main/resources/plugin.yml]: In the case that this ever happens, this doesn't hurt. Not that it will, but let's pretend it might.

**Comment by Sauilitired** [comment]
My opinions on this, copied and pasted from Discord:
> The disable time thing looks very similar to the disabling of titles, but for plot time. I think it’s a fair change to implement, as it doesn’t break anything
It simply allows players to disable their time getting updated when entering a plot. This is useful for many reasons, but I can see players using shaders appreciating it even more, as playing in complete darkness isn’t always a lot of fun

**Comment by jewome62** [comment]
Yes, I confirm, that's for people who don't want theirs screen flashes with day/night when they fly over many plot.

**Comment by jewome62** [comment]
@MattBDev The goal is let user (with permission to do it) to do not suffer change of aggressive change of time
- As admin and moderator who are often on server
- visually impaired
- for shaders

My work is currently tested on my server without problem

**Comment by MattBDev** [comment]
If you can confirm it works, I won't stand in the way of this being merged.
I'll leave this to @N0tMyFaultOG to either close this or merge it.

**Comment by jewome62** [comment]
Wiki updated
Thanks

**Pull request merged**